index.js

```diff
@@ -115,19 +125,115 @@ function parse(options) {
 }
 
 /**
-* @param  {Object} 					 options [posthtml options]
-* @param  {Array | Function} plugins [array of plugins to apply or function, which will be called with from option]
-* @param  {String}           from    [path to the processing file]
-* @param  {Object} 					 content [posthtml tree to process]
-* @param  {Array}            prepend [array of plugins to process before plugins param]
-* @return {Object}                   [processed poshtml tree]
-*/
+ * @param  {Object} 					 options [posthtml options]
+ * @param  {Array | Function} plugins [array of plugins to apply or function, which will be called with from option]
+ * @param  {String}           from    [path to the processing file]
+ * @param  {Object} 					 content [posthtml tree to process]
+ * @param  {Array}            prepend [array of plugins to process before plugins param]
+ * @return {Object}                   [processed poshtml tree]
+ */
 function processWithPostHtml(options, plugins, from, content, prepend) {
   return posthtml((prepend || []).concat(
     typeof plugins === 'function' ? plugins(from) : plugins
   )).process(render(content), options).then(result => result.tree);
 }
 
+/**
+ * Set href for custom tag
+ *
+ * @param  {Object} node [posthtml element object]
+ * @param  {Object} options [posthtml options]
+ * @return {void}
+ */
+function setCustomTagHref(node, options) {
+  if (!node.attrs) {
+    node.attrs = {};
+  }
+
+  const {tag} = node;
+
+  // Get module filename from tag name by removing "x-"
+  //  and replacing dot "." with slash "/" and appending extension
+  const customTagFile = tag
+    .replace(options.customTagPrefix, '')
+    .split('.')
+    .join('/')
+    .concat('.', options.customTagExtension);
```

Comment:
all these `.`, `/` values must be converted to a constant and consider for `\` OS features

Reply:
I was not sure if add folder separator in options so for now is as constant. I could not imagine another separator for folder inside tag name. 

I added only namespace separator in options.